### DISCUSSION TITLE
storage classes rules

### DISCUSSION THREAD

**Discussion by dlednik** [moosefs/moosefs#414] [Q&A]
I was wondering if it is possible for mfs to decide what storage class to store files on based on its size?

For example lets say all files smaller than 16MB in size to go on SSD storage and the rest on HDD storage.
Would that be possible to define?

**Comment by jkiebzak** [comment]
+1

**Reply by dlednik** [reply]
@jkiebzak you can maybe also give thumbs up now in the opened feature request 👍

**Comment by chogata** [comment]
No, currently this is not possible in MooseFS, but it's an interesting idea. We will add it to our feature request list.

**Comment by dlednik** [comment]
Can we have a link to the feature link to track it?

**Comment by chogata** [comment]
Our feature request list is not public, so no, it's not possible to track it.

**Comment by chogata** [comment]
Okay, I have re-read what I wrote and I think my statement may be misunderstood :) Our internal tool for tracking progress on features is, well... internal ;) so it's not possible to track progress. However, features requested by community are available here, so if you want to know when your request has been completed, please, head to the issues section, create new "feature request" issue. When it is completed, it will be marked so and you will be notified.

**Comment by dlednik** [comment]
Ok, I'll create a public feature request here on git and you can link it to your internal tool :)

**Comment by dlednik** [comment]
Feature request https://github.com/moosefs/moosefs/issues/415
